be/src/storage/rowset/segment.h

```diff
@@ -267,6 +268,8 @@ class Segment : public std::enable_shared_from_this<Segment> {
 
     std::shared_ptr<FileSystem> _fs;
     std::string _fname;
+    // -1 indicates tablet meta data do not contain the info of segment size
+    int64_t _segment_size;
```

Comment:
Replace the `_fname` and `_segment_size` with a single variable of type `FileInfo`